packages/website/src/pipeline/stages/generate-pages/index.ts

```diff
@@ -1,15 +1,191 @@
 import createStage from '../make-pipline-stage';
+import {
+  generatePackageDocPage,
+  generateProjectDocPage,
+  generateDocsHomePage,
+  generateDocumentsMainPage,
+  generateExamplesHomePage,
+  generateExamplePage,
+  generateChangelogPage,
+  generateHomePage,
+  // @ts-ignore: Implicit any
+} from './page-writers';
 
-interface StageInput {}
+interface GenericPage {
+  // The path this page should appear at in the finished website
+  websitePath: string;
+  // Additional data used by the page
+  pageData: object;
```

Comment:
Would it be worthwhile to add a type parameter to GenericPage and use that here?

Reply:
I did consider this, but I think it's not worth doing until we add types to the code in page-writers.js